pkg/plugin/admissioncontrollers/webhook/admission_controller.go

```diff
@@ -125,20 +129,23 @@ func updateSchedulerName(patch []patchOperation) []patchOperation {
 	})
 }
 
-// the generated ID is using [PodName]_[Timestamp] naming convention.
-// the generated ID should be unique even the pod name is same (differ on nano time),
+// generate appID based on the namespace value,
 // and the max length of the ID is 63 chars.
-func generateAppID(podName string) string {
-	timestamp := strconv.FormatInt(time.Now().UnixNano(), 10)
-	generatedID := fmt.Sprintf("%.43s_%.19s", podName, timestamp)
-	return generatedID
+func generateAppID(namespace string) string {
```

Comment:
Do we want to keep both options? One based on podName and the other based on namespace?
We have a config object and can then switch giving us the choice between namespace based and just on pod name. Seems a waste to throw away that option.

Reply:
That will be a bit hard to accomplish. We don't have an option to config the admission controller's behavior today.